webcompat/form.py

```diff
@@ -270,9 +253,16 @@ class FormWizard(IssueForm):
         [InputRequired(message=radio_message)],
         choices=browser_choices
     )
+    # Bots Trap
```

Comment:
I think this comment is in the wrong spot. A few lines above is `username`, which is the dummy field. `username_pattern` is used as validation for the `contact` field (which is the optional non-auth github username field). Maybe renaming it to something like `github_username_pattern` would make it less confusing.

Reply:
nice catch I had done it correctly at first. And messed up when reordering.

Reply:
done